slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -0,0 +1,343 @@
+---
+title: "RAP for managers - basics and principles"
+author: "Statistics Development Team"
+institute: Department for Education
+format:
+  revealjs: 
+    theme: black
+    highlight-style: breezedark
+    code-overflow: wrap
+    include-in-header: fontawesome.html
+    width: 120%
+    height: 120%
+    logo: DfE_logo_landscape.png
+    scrollable: true
+    standalone: true
+---
+
+# Why am I here?
+
+This session is designed to cover some of the basic principles of RAP from the perspective of a manager, and to give you a chance to ask any questions that you might have.
+
+<br>
+
+Our main aims are to:
+
+- Avoid managers feeling "left behind" and lacking the confidence to effectively support their teams in achieving best practice
+- Ensure that managers have skill levels at which they can effectively collaborate with and advise their peers and teams
+- Be confident that managers know where to go for RAP advice and support if they cannot provide it directly themselves
+
+
+
+# Do I really need technical skills as a manager? 
+
+<br>
+
+We understand that many managers won't be doing as much of the hands-on coding or technical work, but it's important to have strong enough skill levels in order to successfully support your teams to achieve their potential, or to cover during absence. 
+
+<br>
+
+Applying elements of RAP will also make your lives easier if there is staff turnover or long-term absence, and you have to step in to cover. 
+
+# Intro quiz
+
+::: {.column width="40%"}
+:::
+
+::: {.column width="20%"}
+Menti:
+
+![](images/mentimeter_qr_code.png){fig-align="center"}
+:::
+
+::: {.column width="40%"}
+:::
+
+
+# Q&A - what do you want to know?
+
+::: {.column width="40%"}
+:::
+
+::: {.column width="20%"}
+Slido:
+
+![](images/RFM-slido.png){fig-align="center"}
+:::
+
+::: {.column width="40%"}
+:::
+
+# What is RAP?
+
+::: {.fragment .fade-up}
+<span style="color:#f67400;">**R**</span>eproducible <span style="color:#f67400;">**A**</span>nalytical <span style="color:#f67400;">**P**</span>ipelines are simply about using automation to our advantage when analysing data, and ensuring we are transparent when we do!
+:::
+
+::: {.fragment .fade-up}
+We already have *analytical pipelines* and have done for many years. The aim of RAP is to:
+:::
+
+::: {.fragment .fade-up}
+-   Automate all of the parts that we can
+:::
+
+::: {.fragment .fade-up}
+-   Increase efficiency and accuracy
+:::
+
+::: {.fragment .fade-up}
+-   Create a clear audit trail to allow analysis to be easily reused if needed
```

Comment:
'where' rather than 'if'? Like more often than not a reusability consideration is helpful so it's worth the effort